examples/tutorial/c/tut_pool.c

```diff
@@ -0,0 +1,40 @@
+//////////////////////////////////////////////////////////////////////////////
+// Copyright (c) 2018, Lawrence Livermore National Security, LLC.
+// Produced at the Lawrence Livermore National Laboratory
+//
+// Created by David Beckingsale, david@llnl.gov
+// LLNL-CODE-747640
+//
+// All rights reserved.
+//
+// This file is part of Umpire.
+//
+// For details, see https://github.com/LLNL/Umpire
+// Please also see the LICENSE file for MIT license.
+//////////////////////////////////////////////////////////////////////////////
+#include "umpire/interface/umpire.h"
+
+#define SIZE 1024
+
+int main(int argc, char** argv)
+{
+  umpire_resourcemanager rm;
+  umpire_resourcemanager_get_instance(&rm);
+
+  umpire_allocator allocator;
+  umpire_resourcemanager_get_allocator_by_name(&rm, "HOST", &allocator);
+
+  umpire_allocator pool;
+  umpire_resourcemanager_make_pool_allocator(&rm, "pool", allocator, 1024*512, 512, &pool);
```

Comment:
Do we need to add a comment describing the arguments?  Or is the definition for this function (and other C API functions documented elsewhere?

Reply:
I will do that.